### ISSUE TITLE
question about the method collecting test set for street

### ISSUE THREAD

**Issue by Yang-Xijie** [city-super/MatrixCity#3]
Thanks for the great work!

I have a question about `page 6 > 4.1 Datasets and Metrics > MatrixCity benchmark`:

> test set is collected... For street data, the yaw direction randoms from 0° to 90° and every camera location captures 5 images, whose pitch and roll direction keep the same with the training set.

I check the test folder (eg. `/small_city/street/test/small_city_road_down_test`).

It should be correct that the pitch direction is 0° (horizontally) or 90° (vertically, face to sky) and the roll direction is 0°, which are the same as the six perspective cameras in the training set.

However, I do not understand `the yaw direction randoms from 0° to 90°`. Do you mean a fanshaped 1/4 circle field of view from front to right? Or do you want to express a fanshaped 1/2 circle field of view which should be `the yaw direction randoms from -90° to 90°`? Or simply `the yaw direction randoms from 0° to 360°`?

Besides, I am curious about the reason you didn't use the pitch direction from -90° to 90° to make the views in the test folder *more novel*...

**Comment by Yang-Xijie** [comment]
Here "the yaw direction randoms from 0° to 90°" means that four cameras (with the pitch direction 0°) all have a random yaw direction from 0° to 90°, namely, 0-90°, 90-180°, 180-270°, and 270-360°. So it is identical with "the yaw direction randoms from 0° to 360°".
